### PR TITLE
DAOS-6909 test: Online Drain Testing (More tests)

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag-hw-medium: pr,hw,medium,ib2 online_drain

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

File: src/tests/ftest/osa/osa_online_drain.py [hunk 1]
```diff
@@ -111,23 +107,92 @@ def run_online_drain_test(self, num_pool):
...
+        """
+        self.log.info("Online Drain : Oclass")
+        for oclass in self.test_oclass:
+            self.run_online_drain_test(1, oclass=oclass)
+
+    @skipForTicket("DAOS-7289")
+    def test_osa_online_drain_with_aggregation(self):
+        """Test ID: DAOS-6909
+        Test Description: Validate Online drain with different
+        object class.
```
Comment (shimizukko) [inline src/tests/ftest/osa/osa_online_drain.py:174]:
```suggestion
        Test Description: Validate Online drain with aggregation.
```
typo?

File: src/tests/ftest/osa/osa_online_drain.yaml [hunk 2]
```diff
@@ -42,33 +44,67 @@ server_config:
...
+  iorflags:
+    write_flags: "-w -F -k -G 1"
+    read_flags: "-F -r -R -k -G 1"
+    api: DFS
+    dfs_oclass: RP_2G4
+    dfs_dir_oclass: RP_2G4
+  ior_test_sequence:
     #   - [scmsize, nvmesize, transfersize, blocksize, PASS/FAIL(Expected) ]
     #    The values are set to be in the multiples of 10.
     #    Values are appx GB.
```
Comment (shimizukko) [inline src/tests/ftest/osa/osa_online_drain.yaml:75]: Not sure what you mean by "appx GB". You specified everything in bytes.

File: src/tests/ftest/osa/osa_online_drain.py [hunk 3]
```diff
@@ -28,28 +27,30 @@ def setUp(self):
         """Set up for test case."""
         super().setUp()
         self.dmg_command = self.get_dmg_command()
-        self.ior_flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
-        self.ior_apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        self.daos_command = DaosCommand(self.bin)
```
Comment (shimizukko) [inline src/tests/ftest/osa/osa_online_drain.py:30]:
```suggestion
        self.daos_command = self.get_daos_command()
```
I believe you could do this.

**Review by shimizukko — APPROVED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Comment by sylviachanoiyee** [comment]
@rpadma2 , could you create ticket to re-enable the tests and have them block DAOS-7289? Thanks!

**Pull request merged**